src/current/_includes/releases/v23.2/v23.2.0-alpha.1.md

```diff
@@ -906,3 +968,17 @@ This release includes 3116 merged PRs by 122 authors.
 [fc5920459]: https://github.com/cockroachdb/cockroach/commit/fc5920459
 [fedb692ae]: https://github.com/cockroachdb/cockroach/commit/fedb692ae
 [fef97d93c]: https://github.com/cockroachdb/cockroach/commit/fef97d93c
+{% comment %}Only in new hash{% endcomment %}
```

Comment:
To delete?

Reply:
Yeah before merging. This is to help reviewers and @celiala to verify that I got all the relevant commits for the updated hash.